.changelog/11891.txt

```diff
@@ -0,0 +1,3 @@
+```bug
```

Comment:
i think this needs `release-note:` prefix?
https://github.com/hashicorp/go-changelog/blob/main/note.go#L21

Reply:
Serves me right for not copying a previous one and doing it from memory 😀